app/src/main/java/com/duckduckgo/app/browser/customtabs/CustomTabActivity.kt

```diff
@@ -48,6 +48,7 @@ class CustomTabActivity : DuckDuckGoActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        viewModel.onShowCustomTab()
```

Comment:
Is it still desired to keep the `viewModel.onShowCustomTab()` call in `onStart` too as well as `onCreate`?

Reply:
Yes, in case the tab is put in background and brought back to foreground.